src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
1) What's running at 50Hz? I show Xplane9 at 120Hz.
2) Does FlightGear talk to QGC at 400Hz?  (and how do I reproduce your 400Hz HIL test?)
3) Where does the sensor noise get injected when QGC is bypassed?
4) How do I run HIL without QGC?